Example/WalletApp/ApplicationLayer/AppDelegate.swift

```diff
@@ -24,14 +33,16 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
       _ application: UIApplication,
       didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data
     ) {
-        Task(priority: .high) {
-            // Use pasteboard for testing purposes
-            let tokenParts = deviceToken.map { data in String(format: "%02.2hhx", data) }
-            let token = tokenParts.joined()
-            let pasteboard = UIPasteboard.general
-            pasteboard.string = token
-            try await Push.wallet.register(deviceToken: deviceToken)
-        }
+        
+       // Commenting this out as it breaks UI tests that copy/paste URI
```

Comment:
Question for reviewers...
Not sure how much this is used and whether I should keep it and devise a workaround.